Tasks/AppStoreRelease/task.json

```diff
@@ -13,8 +13,8 @@
     "demands": [ "xcode" ],
     "version": {
         "Major": "1",
-        "Minor": "178",
-        "Patch": "4"
+        "Minor": "182",
```

Comment:
Could you please change task version according current sprint?

Reply:
Done, thanks!